app/assets/javascripts/discourse/app/controllers/composer.js

```diff
@@ -204,14 +204,17 @@ export default Controller.extend({
 
   @discourseComputed("model.canEditTitle", "model.creatingPrivateMessage")
   canEditTags(canEditTitle, creatingPrivateMessage) {
-    if (creatingPrivateMessage && (this.site.mobileView || !this.isStaffUser)) {
+    if (creatingPrivateMessage && this.site.mobileView) {
```

Comment:
Does this mean you can never edit tags on mobile?

Reply:
Yes, that is correct. I believe it is because there is not a lot of space to show the title selector and tag selector. I tried removing this condition and the content was really crowded.